gen-windows.bat

```diff
@@ -1,4 +1,5 @@
 @echo off
+title Generate project solution VS
```

Comment:
remove, not part of this PR

Reply:
is small change, i no like create pr for very small things like that is not important